jenkins/vulnerability-scan/whitesource-scan.jenkinsfile

```diff
@@ -41,8 +41,13 @@ pipeline {
 
 void scan() {
     git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
-    sh '''
+    sh """
+        curl -SLO https://services.gradle.org/distributions/gradle-6.6.1-bin.zip
```

Comment:
This is a short term temp solution as whitesource is not running correctly with gradle wrapper.

Reply:
Yeah after testing, this is the only way currently working for all repos.

Reply:
Can we install `gradle` in the docker container being used? Installing there would prevent running the install every time the job is built.

Reply:
> Can we install `gradle` in the docker container being used? Installing there would prevent running the install every time the job is built.

We are using the docker container from CI-runner, and I don't think we want to install Gradle on that one because we don't need Gradle to be installed as long as we have Gradle wrapper.